model_one/cli.py

```diff
@@ -1,198 +1,274 @@
-from array import array
+import os
 import json
-from re import M
-from time import time
 import requests
-import os.path
-import urllib.parse
 
-import os
+from enum import Enum
+from functools import wraps
+from typing import Optional, List, Union
+
+from pandas import Series
+
 
 API_KEY = os.environ.get("BEYONDML_API_KEY")
-api_url = 'https://api.beyond.ml'
+MINIMUM_ENTRIES = 32
 
-def _models_api():
-    return '{}/v0/models'.format(api_url)
 
+class BeyondmlModelException(RuntimeError):
+    pass
 
-def _get_auth_header():
-    return {'Authorization': API_KEY}
 
+class BeyondmlAPI():
+    URL: str = "https://api.beyond.ml"
 
-class BeyondmlModel():
-    def __init__(self, model=None):
-        if model is not None:
-            self._id = model['model_name']
-            self._status = model['status']
-            self._model_type = model['model_type']
-        return
-
-    def __repr__(self):
-        return str({'id': self._id, 'status': self._status, 'model_type': self._model_type})
-
-    def __str__(self):
-        return str({'id': self._id, 'status': self._model_type, 'model_type': self._model_type})
-
-    def _api_post_request(self, handler, data):
-        if not self._id:
-            raise Exception('Initialize the model.')
-        headers = _get_auth_header()
-        headers['Content-Type'] = 'application/json'
-        url = '{}/{}/{}'.format(_models_api(), self._id, handler)
-        r = requests.post(url, headers=headers, data=data)
-        if r.status_code != 200:
-            raise Exception(r.text)
-        self._status = r.json()['status']
-        return r.json()
+    V0: dict = {
```

Comment:
Here we have all the methods & paths to the available endpoints. They can be freely changed while key referring to them stay the same

Reply:
Hope i did not miss something and all the methods are correct